examples/03_renderable/main.rs

```diff
@@ -31,17 +32,18 @@ struct DemoState {
     point_light: bool,
     directional_light: bool,
     camera_angle: f32,
-    #[allow(dead_code)]
-    pipeline_forward: bool, // TODO
+    #[allow(dead_code)] pipeline_forward: bool, // TODO
```

Comment:
Could we change that in the rustfmt config?

Reply:
Why?  I kind of like it this way.  It communicates the impact of the #[] more clearly.

Reply:
But shouldn't the focus be on the field name and its type rather than attributes? And I also had a case where the line got super long because of that formatting and thus wasn't very nice to read.

Reply:
I don't feel this really removes any visibility of that information, but I also have no strong feelings on formatting other than "it shouldn't be absolutely awful to read and write".  So sure I'll add it to the config.